vm/src/builtins/float.rs

```diff
@@ -546,50 +545,29 @@ impl Hashable for PyFloat {
 impl AsNumber for PyFloat {
     fn as_number() -> &'static PyNumberMethods {
         static AS_NUMBER: Lazy<PyNumberMethods> = Lazy::new(|| PyNumberMethods {
```

Comment:
as PyNumberMethods is fully immutable, we should able to remove the Lazy<> things